pkg/k8shandler/logging.go

```diff
@@ -14,10 +15,15 @@ import (
 
 type ClusterLogging struct {
 	*logging.ClusterLogging
+	Runtime *runtime.OperatorRuntime
```

Comment:
we're using a runtime?

Reply:
maybe bad naming choice but its just the interface to allow mocking the api calls